query-node/build.sh

```diff
@@ -11,19 +11,7 @@ set +a
 
 yarn clean
 
-# We generate the code for each service separately to be able to specify
-# separate database names.
-
-# Build indexer customizing DB name
-DB_NAME=${INDEXER_DB_NAME} yarn codegen:indexer
-
-# Build graphql-server customizing DB name
-DB_NAME=${PROCESSOR_DB_NAME} yarn codegen:server
-
-# We run yarn again to ensure processor and indexer dependencies are installed
-# and are inline with root workspace resolutions
-yarn
```

Comment:
We still need another `yarn` step after codegen to properly install the dependencies after the generated/graphql-server pacakge is generated.

Reply:
perhaps:
```
yarn codegen:noinstall
yarn
```
There is also some problem coming from installing dependencies in particular the monorepo's husky configuration is being overridden by the husky configuration of generated/graphql-server. Is there some option to skip running the husky bootstrapping somehow.

Reply:
`generated/graphql-server/package.json` has this   

```JSON
"husky": {
    "hooks": {
      "pre-commit": "yarn run config:dev && lint-staged && tsc -p ./tsconfig.json && yarn test"
    }
  },
```

is it this what you want to get rid of? if yes, it is possible and would require a new hydra-cli release.

Reply:
Well that is what is interfering yes, but I don't want to get rid of it per say, I'm sure it is useful for projects that are checking in the graphql-server into their repo. I think maybe if we fix this https://github.com/Joystream/joystream/pull/2167#discussion_r586852488 it might no longer be an issue?